packages/subgraph/schema.graphql

```diff
@@ -280,6 +280,22 @@ interface Proposal {
   metadata: String
   actions: [Action!]! @derivedFrom(field: "proposal")
   allowFailureMap: BigInt!
+  failureMap: BigInt
+  executed: Boolean!
+  createdAt: BigInt!
+  startDate: BigInt!
+  endDate: BigInt!
+  executionTxHash: Bytes
+}
```

Comment:
may be it is good opertunity to rename `Proposal` to `IProposal`.

Reply:
done